packages/ui-components/style/icons/output/collapse.svg

```diff
@@ -0,0 +1,10 @@
+<svg height="24px" viewBox="0 0 8.5 10.5" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Our icon must be squared. Could you please update the SVG and also should have the same default size `width="16"`?

Would you be also able to minimify the icon by merging the transformation with the path? If you don't know how to, I use the `jock.svg` extension for VS Code.

Reply:
> > > Hi @brijsiyag , it would be best if the icon does not change size. I would suggest using a similar size to the other icons on the interface, probably the size that is used for the 1-line output. And for single line outputs I think we should hide the icon, as there is no scrolling available.
> > 
> > 
> > Should we set it to 16px ? I think 16px will be too small if we see figma design. I'm setting height 24px (Sidebar icon's size) for now, tell me if have to alter that.
> 
> Let's try that and see how it feels. 24px is also a good icon height for desktop application so it might be ok!

This is the comment from @GabrielaVives. Let me post a look of both here for comparison, 

<img height="360" alt="Screenshot 2023-06-16 at 12 18 42 AM" src="https://github.com/jupyterlab/jupyterlab/assets/71015892/30a67b92-abbc-45f8-a1b3-63eb7d918d90">
<img height="360" alt="Screenshot 2023-06-16 at 12 19 22 AM" src="https://github.com/jupyterlab/jupyterlab/assets/71015892/5575589c-43ac-4409-adda-ad69fb46e519">

@fcollonval PTAL and let me know which size we should use.

Reply:
and thanks for the extension suggestion, I will do update suggested changes.

Reply:
After seeing them side-by-side I would vote for 16px. Another point is that we can also resize icons using `LabIcon` attributes, but regardless of the size in UI, our SVG icon files are conventionally all 16px wide on 24x24 grid for consistency.

Reply:
Same as @krassowski , I think the 16px looks consistent with the visual information surrounding this button. The larger version feels out of place.

Reply:
Yeah agreed, I've updated the PR with width="16".